src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -279,7 +279,7 @@ public void copyFile(File file) {
                     // TODO: Make it work for RemoteJenkinsController like in Unix (below)
                     throw new AssumptionViolatedException("Copying files in Windows is only supported if a LocalController is in use. Test will be skipped.");
                 }
-                addBatchStep("xcopy " + file.getAbsolutePath() + " %cd% /E");
+                addBatchStep("xcopy " + file.getAbsolutePath() + " %cd% /E /Y");
```

Comment:
Would it make sense to extract ` /E /Y` as constant and reuse it as kind of `WINDOWS_XCOPY_OPTIONS` for the other occurrence in line as well? Or add a parameterized `xcopy` method that can be used in both places?

Reply:
a method has been added